Index: .env
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.env b/.env
new file mode 100644
--- /dev/null	(date 1787812162506)
+++ b/.env	(date 1787812162506)
@@ -0,0 +1,5 @@
+BOT_TOKEN=8700221693:AAFlb9vsaCLoJMgLvcqTIPU98pp-LQQQ3gY
+DATABASE_URL=sqlite+aiosqlite:///./box_game.db
+ADMIN_IDS=6261056411
+COOLDOWN_SECONDS=120
+WINNING_BOXES=25:10000,54:7000,73:5000,88:20000,96:50000
Index: .idea/.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.idea/.gitignore b/.idea/.gitignore
new file mode 100644
--- /dev/null	(date 1787811787171)
+++ b/.idea/.gitignore	(date 1787811787171)
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
+/workspace.xml
+# Editor-based HTTP Client requests
+/httpRequests/
